src/main/kotlin/no/nav/helse/spenn/oppdrag/OppdragSkjemaConstants.kt

```diff
@@ -35,6 +33,15 @@ class OppdragSkjemaConstants {
 
 }
 
+enum class FagOmraadekode(val kode: String) {
+    SP("SP"), //bruker er mottaker
+    SPREF("SPREF") // refusjon til arbeidsgiver
```

Comment:
istedenfor kommentarer så burde enum-navnene reflektert hva det er

Reply:
Jo, men dette er kode som bare gir mening for OS sitt domain, derfor tenker at jeg at vi bør bruker det som står det.. så det ikke blir gjemt fra utvikleren. Jeg håndterer det som et "navn"

Reply:
men vi må også forstå OS sitt domene for å kunne integrere mot det også, tenker jeg. Da syns jeg noe sånn leser bedre:
```
enum class FagOmraadekode {
    Sykepenger("SP"),
    Sykepengerefusjon("SPREF")
}
```